.github/workflows/ci.yml

```diff
@@ -78,31 +80,9 @@ jobs:
         id: get_tag_version
         run: echo TAG_VERSION=${GITHUB_REF/refs\/tags\//} >> "$GITHUB_OUTPUT"
 
-      - name: Publish deno.land/x
-        uses: denoland/publish-folder@82ce065074e7174baf444332c4b7c40869a4909a
+      - name: Publish JSR
         if: contains(matrix.os, 'ubuntu') && startsWith(github.ref, 'refs/tags/')
-        with:
-          folder: js
-          branch: deno_registry
-          tag: deno/${{ steps.get_tag_version.outputs.TAG_VERSION }}
-          token: ${{ secrets.DENOBOT_PAT }}
-          git-user-name: denobot
-          git-user-email: denobot@users.noreply.github.com
-
-        # NPM Distribution
-      - name: Setup node
-        uses: actions/setup-node@v2
-        if: contains(matrix.os, 'ubuntu')
-        with:
-          node-version: '18.x'
-          registry-url: 'https://registry.npmjs.org'
-
-      - name: Build npm
-        if: contains(matrix.os, 'ubuntu')
-        run: deno task build:npm ${{steps.get_tag_version.outputs.TAG_VERSION}}
-
-      - name: npm publish
-        if: contains(matrix.os, 'ubuntu') && startsWith(github.ref, 'refs/tags/')
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: cd npm && npm publish
```

Comment:
I need dnt to work with jsr in order to use this, but I need this on jsr in order to upgrade dnt, so removing this. Also, it's been broken for months and nobody seems to be using it, so removing.